themis/regulations/apple-crypto-regulations.md

```diff
@@ -1,9 +1,9 @@
 ---
 weight: 1
-title:  App Store requirements
+title: App Store requirements
 ---
 
-# Apple encryption export regulations for Themis
+# US crypto export regulations for Themis when submitting to App Store
```

Comment:
Similarly, I believe there should be only one `<h1>` header on the page which serves as page title. Others should be structured under it. Plus, multiline headers don't look that great.

```suggestion
# US crypto export regulations for Themis

## Submitting apps to App Store
```